Userland/Libraries/LibGL/Lighting.cpp

```diff
@@ -0,0 +1,541 @@
+/*
+ * Copyright (c) 2021, Jesse Buhagiar <jooster669@gmail.com>
+ * Copyright (c) 2021, Stephan Unverwerth <s.unverwerth@serenityos.org>
+ * Copyright (c) 2022, Jelle Raaijmakers <jelle@gmta.nl>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Debug.h>
+#include <LibGL/GLContext.h>
+
+extern GL::GLContext* g_gl_context;
```

Comment:
I think all those extern declarations can be removed.

Reply:
They can! And they are :-)